<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigGuiClass</class>
 <widget class="QWidget" name="ConfigGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        QPushButton:!hover
        {
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
        border: none;
        border-radius:5px;
        color: rgb(255, 255, 255);
        font: 75 12pt &quot;黑体&quot;;
        }
        QPushButton:hover
        {
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
        border: none;
        border-radius:5px;
        color: rgb(255, 255, 255);
        font: 75 12pt &quot;黑体&quot;;
        }

        QLabel
        {
        color: rgb(255, 255, 255);
        font: 75 10pt &quot;黑体&quot;;
        }

        #back
        {
        border-radius:5px;
        background-color: rgb(53, 53, 53);
        }

        #closeButton{
        font: 150 14pt &quot;等线&quot;;
        color: rgb(255, 255, 255);
        background-color: rgb(53, 53, 53);
        }

        #title
        {
        color: rgb(0, 170, 255);
        font: 75 12pt &quot;黑体&quot;;
        }
        #line
        {
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(28, 28,28, 255), stop:1 rgba(68, 68, 68, 255));
        }
      </string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XMS微服务配置管理中心</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QWidget" name="line" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1280</width>
      <height>2</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>691</width>
      <height>641</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>微服务名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>端口</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>dir</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>8080</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_ip_edit">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="server_port_box">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>20019</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>配置中心IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>配置中心端口</string>
    </property>
   </widget>
   <widget class="QListWidget" name="log_list_Widget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>331</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_config_Button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新增配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="config_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1251</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>slotRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1218</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>1288</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_config_Button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>slotAddConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1201</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>1303</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotRefresh()</slot>
  <slot>slotAddConfig()</slot>
 </slots>
</ui>
